core/CMakeLists.txt

```diff
@@ -80,6 +80,31 @@ if (AARCH64)
     ${aarch64_exported_opcodes} VERBATIM)
 endif ()
 
+if (RISCV64)
```

Comment:
I assume the codec builds successfully now. If so, can you add it to the PR description?

Reply:
Yes, however one CMake change got mixed up in #5670 so I'll have to move it here.